meeting/models.py

```diff
@@ -42,4 +42,13 @@ def create_user_profile(sender, instance, created, **kwargs):
 @receiver(post_save, sender=User)
 def save_user_profile(sender, instance, **kwargs):
     instance.profile.save()
-    
\ No newline at end of file
+    
+''' Time availability '''
+class TimeAvailability(models.Model):
+    start_time = models.DateTimeField()
+    end_time = models.DateTimeField()
+    meeting = models.IntegerField()
+    # meeting = models.ForeignKey(Meeting, on_delete=models.CASCADE)
+    # user = models.ForeignKey(User, on_delete=models.CASCADE)

```

Comment:
When you go to add "user", consider using the "Profile" class, not the "User" class.